### PR TITLE
feat: mobile color normalization

### PR DESCRIPTION
**Pull request by GleidsonDaniel**
## Proposed changes

Changes the color tags to use the standard that is currently used on the web, normalizing the mobile app's themes and color usage.

## How to test or reproduce

- From within the app, change the themes, test all possible combinations and see if you notice any discrepancies in the colors.

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] Improvement - BREAKING CHANGE
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [x] I have added necessary documentation (if applicable)
- [x] Any dependent changes have been merged and published in downstream modules

## Further comments

Jira: [NATIVE-506]

[NATIVE-506]: https://rocketchat.atlassian.net/browse/NATIVE-506?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/containers/RoomItem/Actions.tsx [hunk 1]
```diff
@@ -21,6 +21,8 @@ import I18n from '../../i18n';
 const CONDENSED_ICON_SIZE = 24;
 const EXPANDED_ICON_SIZE = 28;
 
+const starColor = '#f5d100';
```
Comment (diegolmello) [inline app/containers/RoomItem/Actions.tsx:24]: This feels wrong.
There might be no reason to make it part of any theme, but shouldn't we unify colors?
I wonder how web deals with the fav icon.

File: app/containers/TextInput/FormTextInput.tsx [hunk 2]
```diff
@@ -28,8 +28,8 @@ const styles = StyleSheet.create({
 		fontSize: 16,
 		paddingHorizontal: 16,
 		paddingVertical: 10,
-		borderWidth: 1,
-		borderRadius: 4
+		borderWidth: 2,
+		borderRadius: 2
```
Comment (diegolmello) [inline app/containers/TextInput/FormTextInput.tsx:32]: We changed this a while ago intentionally.
Feels weird/old to have such a thick border width.
Can you create a thread with Jose about it and mention me? 😄

**Review by diegolmello — APPROVED** [review]

File: app/containers/RoomItem/Actions.tsx (hunk 1 above)
Comment (GleidsonDaniel) [reply app/containers/RoomItem/Actions.tsx:24]: It has no color

**Review by diegolmello — COMMENTED** [review]
### general

- TextInput has no border radius
- ServerDropdown's "create workspace" shouldn't have border radius

### light

- quotes look like links (tint)
- mention of other people isn't highlighting enough

### dark

- LastMessage has not enough contrast
- react-navigation adds a very light line below headers (I'm pretty sure it is from the lib)  
- roles don't have background
- Some RoomItem actions are hard to read (dark background and dark text)
- DirectoryView dropdown shouldn't be blue imo
- ServerDropdown has some dark backgrounds with blue text
- E2EE banner is blue background with dark font. Hard to read.
- Button as primary disabled is blue background with dark font. It's hard to read, even though it's disabled.
- Logout has a low contrast
- Maybe the blue from "Apply your certificate" could be used more?

### black

- Headers could be black
- Message body should be less bright than username
- NotificationsPreferencesView is weird. It gives emphasis to the wrong things. I think it's because the listitems are black, but the background is lighter.
- Switches should be lighter
- Troubleshooting bell disabled is gray
image
- Composer's Autocomplete is hard to read

File: app/lib/services/connect.ts [hunk 3]
```diff
@@ -270,6 +275,8 @@ function connect({ server, logoutOnError = false }: { server: string; logoutOnEr
 			})
 		);
 
+		logoutListener = sdk.current.onStreamData('stream-force-logout', () => store.dispatch(logout(true)));
```
Comment (diegolmello) [inline app/lib/services/connect.ts:278]: This is from a different PR. Can you remove it?

**Review by diegolmello — APPROVED** [review]

File: app/lib/services/connect.ts (hunk 3 above)
Comment (GleidsonDaniel) [reply app/lib/services/connect.ts:278]: kek

**Pull request merged**